src/test/java/net/quedex/api/user/UserMessageSenderTest.java

```diff
@@ -364,6 +364,87 @@ public void sendTimeTriggeredBatchUpdate() throws Exception {
         );
     }
 
+    @Test
+    public void sendTimeTriggeredBatchUpdateWithExecutionStartMovedOnly() throws Exception {
+        // given
+        final long timerId = 1L;
+        final long executionStartTimestamp = 200L;
+
+        // when
+        sender.sendTimeTriggeredBatchUpdate(timerId, executionStartTimestamp, null, null);
+
+        // then
+        verify(wsClient, timeout(100)).send(
+            "{" +
+                "\"type\":\"update_timer\"," +
+                "\"timer_id\":1," +
+                "\"new_execution_start_timestamp\":200," +
+                "\"new_execution_expiration_timestamp\":null," +
+                "\"account_id\":1234," +
+                "\"nonce\":1," +
+                "\"nonce_group\":5," +
+                "\"new_command\":null" +
+                "}"
+        );
+    }
+
+    @Test
+    public void sendTimeTriggeredBatchUpdateWithUpdatedCommandOnly() throws Exception {
+        // given
+        final long timerId = 1L;
+        final List<OrderSpec> batch = ImmutableList.of(
+            CancelAllOrdersSpec.INSTANCE,
+            new LimitOrderSpec(
+                888L,
+                512,
+                OrderSide.BUY,
+                1500,
+                BigDecimal.valueOf(1L, 8)
+            )
+        );
+
+        // when
+        sender.sendTimeTriggeredBatchUpdate(timerId, null, null, batch);
+
+        // then
+        verify(wsClient, timeout(100)).send(
+            "{" +
+                "\"type\":\"update_timer\"," +
+                "\"timer_id\":1," +
+                "\"new_execution_start_timestamp\":null," +
+                "\"new_execution_expiration_timestamp\":null," +
+                "\"account_id\":1234," +
+                "\"nonce\":1," +
+                "\"nonce_group\":5," +
+                "\"new_command\":{" +
+                "\"type\":\"batch\"," +
+                "\"account_id\":1234," +
+                "\"batch\":[" +
+                "{" +
```

Comment:
Formatting could be better

Reply:
Let's fix when editing this file next time.